app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -380,10 +379,10 @@ public void onClick(View v) {
     }
   }
 
-  public class ProcessJson extends AsyncTask<String, Void, ArrayList<CafeteriaModel>> {
+  public class ProcessJson extends AsyncTask<String, Void, List<EateryModel>> {
```

Comment:
Not my work, but I agree.

Reply:
I'm going to say out-of-scope of this merge as it was a pre-existing codebase decision that is replicated in classes like JsonUtilities. It can be fixed in a future merge, this one has already gotten quite expansive.